src/xcode/ENA/ENA/Source/Services/CoronaTestService/CoronaTestService.swift

```diff
@@ -633,5 +648,64 @@ class CoronaTestService {
 		antigenTestIsOutdated = Date() >= antigenTestOutdatedDate
 	}
 
+	#if DEBUG
+
+	private var mockPCRTest: PCRTest? {
+		if let testResult = mockTestResult(for: .pcr) {
+			return PCRTest(
+				registrationDate: Date(),
+				registrationToken: "asdf",
+				testResult: testResult,
+				finalTestResultReceivedDate: testResult == .pending ? nil : Date(),
+				positiveTestResultWasShown: UserDefaults.standard.string(forKey: "pcrPositiveTestResultWasShown") == "YES",
+				isSubmissionConsentGiven: UserDefaults.standard.string(forKey: "isPCRSubmissionConsentGiven") == "YES",
```

Comment:
Would be cool to have the strings as static constants (and later stored similar to the AppStrings). This could help to keep track on all these parameters and launch arguments.

Reply:
I like that suggestion. That is something I would like to add to the refactoring/enhancement ticket for the exposure submission UI tests that we were just talking about.